java-squid/src/main/java/org/sonar/plugins/java/api/tree/CompilationUnitTree.java

```diff
@@ -34,12 +35,17 @@
 @Beta
 public interface CompilationUnitTree extends Tree {
 
+  @Nullable
+  PackageTree packageDeclaration();
+
   /**
    * @since Java 1.5
    */
+  @Deprecated
   List<AnnotationTree> packageAnnotations();
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1133)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add the missing @deprecated Javadoc tag. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AMissingDeprecatedCheck)